### DISCUSSION TITLE
关于播放列表在分集很多时变得很长

### DISCUSSION THREAD

**Discussion by H3nryM4** [the1812/Bilibili-Evolved#2109] [功能建议 / Ideas]
一个视频有 700 多个分集，展开播放列表后会显示得非常长，很难看到当前分集（比如 300 多集）的title，只能向下滑动去找。
2 种建议：
1. 可以选择将分集title显示在主title下。
2. 只是在title过长时换行显示，不要展开（仍然滚动）。

环境：surface go + ms edge

![image](https://user-images.githubusercontent.com/53041402/127756785-ce5557fd-a194-48bb-8386-b661874f34e7.png)
例如这样，我看到了p241，当我想要查看title时我必须向下滑动很久才能翻到。

**Comment by Geeyun-JY3** [comment]
除第 1 条建议，和我这说的是一个意思吧：https://github.com/the1812/Bilibili-Evolved/issues/2070#issuecomment-886164045

#692
